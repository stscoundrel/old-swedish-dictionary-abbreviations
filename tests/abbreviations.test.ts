import abbreviations, { findWorksAndAuthors } from '../src';

describe('Works & authors abbr mapping tests', () => {
  test('Abbreviations in map have expected content', () => {
    const resultMap = abbreviations.getWorksAndAuthorsMapping();

    // There is little point in testing key => value pairs.
    // Just snapshot it.
    expect(resultMap).toMatchSnapshot();
  });

  test('Finds works & authors in content', () => {
    // From definition of 'perlament'
    const content1 = 'eg. talande; ordväxling, träta, kif. vare thz saka ath drengen giorde nakat perlament i gillet thaa skal husbonden swara fore drengen SO 205 . Jfr parlamente';
    const expected1 = new Map([
      ['SO', 'Skrå-Ordningar. Utg. af G. E. Klemming. 1856.'],
    ]);

    // From definition of 'portpika'
    const content2 = 'flicka som tjänstgör som portvakt. VKU 24 ( 1541) .';
    const expected2 = new Map([
      ['VKU', 'Vadstena klosters Uppbörds- och Utgiftsbok 1539--1570. Utg. af C. Silfverstolpe. 1895.'],
    ]);

    // From definition of 'infödha'
    const content3 = 'L. - part. pret. infödder. 2) medfödd. " mäktogare är indföt onth än owanth got " GU C 20 (hand 2) s. 30. ib. 3) infödd. GU C 20 (hand 2) s. 30. nager jnrikes jnföder man STb 3: 331 (1497).';
    const expected3 = new Map(
      [
        ['GU C 20', 'Latinskt-Svenskt glossarium efter Cod. Ups. C 20. Utg. av E. Neuman. S 1--583. 1918--20, (hand 2) s. 1--169. 1938--42. SFSS.'],
        ['GU', 'Glossarii Latino-Svethici specimen vetustum. E cod. mscr. Bibliothecæ Reg. Acad. Upsal. Diss. Ups. præs. J. H. Schröder. 1845.'],
        ['ST', 'Själens Tröst. Utg. af G. E. Klemming. 1871--73.'],
      ],
    );

    const result1 = findWorksAndAuthors(content1);
    const result2 = findWorksAndAuthors(content2);
    const result3 = findWorksAndAuthors(content3);

    expect(result1).toEqual(expected1);
    expect(result2).toEqual(expected2);
    expect(result3).toEqual(expected3);
  });
});
