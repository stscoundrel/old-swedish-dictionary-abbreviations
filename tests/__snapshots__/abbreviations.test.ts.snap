// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Works & authors abbr mapping tests Abbreviations in map have expected content 1`] = `
Map {
  "A. A." => "Antiquitates Americanae. (E. II.)",
  "Al" => "Konung Alexander. Utg. af G.E. Klemming. 1862.",
  "Ansg" => "Vita Sancti Anscharii per S. Rembertum. Latine et svetice. I Scriptores rerum svecicarum T. 2, sect. 1, s. 173--260.",
  "AS" => "Ur en Antecknares Samlingar. Utg. af G. E. Klemming. 1868--73.",
  "Ber" => "Helige Bernhards Skrifter. Utg. af H. Wieselgren. 1866.",
  "BGD" => "Beskrifning öfver Grefskapet Dal af A. Lignell. 1851--52.",
  "BGG" => "Bröderne af S:t Gertruds Gille i Stockholm 1419--1484. I Småstycken på Forn Svenska, samlade af G. E. Klemming. 1868--81.",
  "Bil" => "Codex Bildstenianus. Se Lg.",
  "Bir" => "Heliga Birgittas Uppenbarelser. Utg. af G. E. Klemming. Bd 1--5. 1857--84. -- Med den vid citat härur stundom förekommande förkortningen: Avt åsyftas de Bd 4, s. 177--185, tryckta egenhändiga utkasten af Birgitta, med Dikt Engelens Diktamen tryckt Bd 4, s. 215--276.",
  "BjR" => "Bjärköa-Rätten. Utg. af C. J. Schlyter. 1844.",
  "BK" => "Margaretæ de Sancta Birgitta Chronicon. Ed. E. Rietz. 1844.",
  "Bo" => "Bonaventuros Betraktelser. Utg. av G. E. Klemning. 1860. SFSS.",
  "BS" => "Bagarnes Skrå. I Småstycken på Forn Svenska.",
  "BSH" => "Bidrag till Skandinaviens historia ur utländska arkiver samlade och utgifna af C. G. Styffe. Del. 1--5. 1859--84.",
  "BtFH" => "Bidrag till Finlands Historia. Utg. genom R. Hausen. 1881--83.",
  "BtRK" => "(Bihang till Rimkrönikorna) Then Andra Delen til Rijm-Crönikorne hörande, innehåller åtskillige Konglige och Furstlige Förlijkningar, Föreningar, etc. Uplagd aff Johan Hadorphio. 1676.",
  "Bu" => "Codex Bureanus. Se Lg.",
  "BYH" => "Samlingar och anteckningar till en beskrifning öfver Ydre Härad i Östergötland, af L. F. Rääf. 1--4. 1856--65.",
  "DD" => "Diplomatarium Dalekarlicum. Urkunder rörande landskapet Dalarne. Utg. af C. G. Kröningssvärd och J. Lidén. Del. 1--3. 1842--46.",
  "Di" => "Sagan om Didrik af Bern. Utg. af G. O. Hyltén-Cavallius. 1850--54.",
  "EG" => "Eriks af Pommern Gårdsrätt. I Småstycken på Forn Svenska.",
  "FH" => "Handlingar till upplysning af Finlands Häfder. Utg. af A. I. Arwidsson. Del 1--9. 1846--57.",
  "Fl" => "Flores och Blanzeflor. Utg. af G. E. Klemming. 1844.",
  "FM" => "Nya källor till Finlands Medeltidshistoria. Utg. af E. Grönblad. 1857.",
  "Fr" => "Hertig Fredrik af Normandie. Utg. af J. A. Ahlstrand. 1853.",
  "Gers Ars" => "Ars moriendi. Gersons lärdom huru man skall dö. 1881.",
  "Gers Frest" => "Om djefvulens frestelse. Af Joh. Gerson. Öfversatt af Ericus Nicolai. 1876.",
  "GO" => "Gamla Ordspråk. Utg. af H. Reuterdahl. 1840.",
  "Gr" => "Legenden om Gregorius af Armenien. Utg. af G. E. Klemming. 1860.",
  "GS" => "Några Gambla Stadgar [såsom bihang till Biärköa-Rätten utg. af J. Hadorph]. 1687.",
  "GU" => "Glossarii Latino-Svethici specimen vetustum. E cod. mscr. Bibliothecæ Reg. Acad. Upsal. Diss. Ups. præs. J. H. Schröder. 1845.",
  "HelsL" => "Helsinge-Lagen. Utg. af C. J. Schlyter. 1844.",
  "HSH" => "Handlingar rörande Skandinaviens historia. 1816 ff.",
  "Iv" => "Herr Ivan Lejonriddaren. Utg. af J. W. Liffman och G. Stephens. 1849.",
  "JP" => "Svensk Järteckens Postilla. Utg. af E. Rietz. 1850.",
  "KL" => "Klosterläsning. Utg. af G. E. Klemming. 1877--78.",
  "Kr LL" => "Konung Christoffers Landslag. Utg. af C. J. Schlyter. 1869.",
  "KS" => "En nyttigh bok om Konnunga Styrilse och Höfdinga. [Utg. af J. T. Bure.] 1634. -- Af de inom parentes tillagda siffrorna utmärker den förra den sida, hvarest citatet återfinnes i Scheffers upplaga, 1669, och den senare motsvarande sida i Geetes, 1878. Då Fragm. tillägges, åsyftar citatet: Ett fragment af Konungastyrelsen, tryckt i Småstycken på Forn Svenska.",
  "LB" => "Läke- och Örte-Böcker. Utg. af G. E. Klemming 1--10. 1883--86.",
  "LfK" => "Skrifter till Läsning för Klosterfolk. Utg. af F. A. Dahlgren. 1875.",
  "Lg" => "Ett Forn-Svenskt Legendarium. Efter gamla handskrifter af G. Stephens. Första bandet. Andra bandet. 1847--58.  -- Särskildt citeras de här aftryckta Codex Bureanus (Bu) och Codex Bildstenianus (Bil), så framt icke senare ändringar i dessa handskrifter afses.",
  "Lg 3" => "Ett Forn-Svenskt Legendarium. Efter gamla handskrifter af G. Stephens. Tredje bandet. 1874.",
  "Lg Manfr" => "Legenda Cardinalis Manfredi. Utg. af E. Rietz i legendæ Suecanæ. 1845.",
  "Lg Mecht" => "Legenda S. Mechtildæ. Utg. af E. Rietz i Vita Gregorii. 1843.",
  "MB" => "Svenska Medeltidens Bibelarbeten. Utg. af G. E. Klemming. Del. 1, 2. 1848--55.",
  "MD" => "Svenska Medeltids dikter och rim. Utg. af G. E. Klemming. 1881--82. SFSS.",
  "MEG" => "Magnus Erikssons Gårdsrätt. I Småstycken på Forn Svenska.",
  "MELL" => "Konung Magnus Erikssons Landslag. Utg. af C. J. Schlyter. 1862.",
  "MK" => "Marias klagan vid korset. I Småstycken på Forn Svenska.",
  "MP" => "Svenska Medeltids-postillor. Utg. af G. E. Klemming. Fortsatta af R. Geete. Del. 3, 4, 5. 1893--1910. SFSS.",
  "OB" => "Omständelig Berättelse huru the svenske af tyskarne blefvo i Stockholm förrådhne oc brände, åhr 1389. I Scriptores rerum svecicarum T. 1, sect. post., s. 212--215.",
  "Pa" => "S. Patriks-Sagan. Utg. af G. Stephens och J. A. Ahlstrand. 1844. Ett inom parentes tillagdt Tung betecknar den i samma band utgivna legenden om Tungulus.",
  "PfN" => "Previlegier för Norberg. I Småstycken på Forn Svenska.",
  "PK" => "Sveriges Krönika (vanligen kallad den prosaiska). I Småstycken på Forn Svenska.",
  "PM" => "Peder Månssons Stridskonst och Stridslag. Utg. af G. O. Hyltén-Cavallius. 1845.",
  "RK" => "Svenska Medeltidens Rim-Krönikor. Utg. af G. E. Klemming. Del. 1--3. 1865--68. Med RK 1 utan vidare tillägg förstås Gamla eller Eriks-Krönikan; då något af de öfvriga i första bandet intagna rimverken anföres, tillägges inom parentes dess titel med begagnande af följande förkortningar: sfgn: fortsättning af Gamla Krönikan för att sammanbinda henne med den nya till ett helt verk.; Albr: Om konung Albrekt: LRK: Lilla Rimkrönikan.",
  "SD" => "Svenskt Diplomatarium. Bd 6 s. 265--584. 1916--21. Bd 8 s. 1--272. 1953.",
  "SD NS" => "(Svenskt Diplomatarium. Ny Serie.) Svenskt Diplomatarium från och med år 1401. Bd 3. Utg. genom C. Silfverstolpe. 1885--1902. -- Bd 4 s. 1--240. Utg. genom K. H. Karlsson. 1903--1904.",
  "SEG" => "Stadga för ett S:t Eriks Gille nära Upsala. I Småstycken på Forn Svenska.",
  "SGG" => "Stadga för ett S:t Görans Gille. I Småstycken på Forn Svenska.",
  "SGGK" => "Stadga för ett S:t Görans Gille vid Kopparberget. I Småstycken på Forn Svenska.",
  "SJ" => "2 Stockholms Stads Jordebok 1474--1498. Utg. genom H. Hildebrad [och L. M. Bååth]. 1889, 1914.",
  "SKG" => "Stadga för S. Karins Gille. I Småstycken på Forn Svenska.",
  "SMG" => "Stadga för ett Jungfru Mariæ Gille. I Småstycken på Forn Svenska.",
  "SML" => "Södermanna-Lagen. Utg. af C. J. Schlyter. 1838.",
  "SmålL" => "Kristnu-Balken af Smålands-Lagen. Utg. af C. J. Schlyter. 1844.",
  "SO" => "Skrå-Ordningar. Utg. af G. E. Klemming. 1856.",
  "SR" => "Upplysningar och Anmärkningar om en för Sveriges Laghistoria vigtig urkund, den förlorade Söderköpings-Rätten. Af G. E. Klemming. I Vitt. Hist, och Ant. Ak. Handl. Del. 25, s. 263--286.",
  "ST" => "Själens Tröst. Utg. af G. E. Klemming. 1871--73.",
  "StadsL" => "Konung Magnus Erikssons Stadslag. Utg. af C. J. Schlyter. 1865.",
  "Su" => "H. Susos Gudeliga Snilles Väckare. Utg. af R. Bergström. 1868--70.",
  "TB" => "(Trögh Bolag) Skogsstadga för mädhal tridhiung i Trögd. I Småstycken på Forn Svenska.",
  "TK" => "Translatio Katarinæ. I Scriptores rerum svecicarum T. 3, sect. post., s. 268--75.",
  "TS" => "Timmermäns Skrå af 1454. I Småstycken på Forn Svenska.",
  "UplL" => "Uplands-Lagen. Utg. af C. J. Schlyter. 1834.",
  "Va" => "Namnlös och Valentin. Utg. af G. E. Klemming. 1846.",
  "VAH" => "Vitterhets Historie och Antiqvitets Akademiens Handlingar.",
  "VGL" => "Vestgöta-Lagen. Utg. af H. S. Collin och C. J. Schlyter. 1827.",
  "VKR" => "Vadstena Kloster-Reglor. Utg. af C. F. Lindström. 1845.",
  "VML" => "Vestmanna-Lagen. Utg. af C. J. Schlyter. 1841.",
  "VNB" => "Vadstena Nunnas Bönbok. Utg. af E. Rietz. 1842.",
  "ÖGL" => "Östgöta-Lagen. Utg. af H. S. Collin och C. J. Schlyter. 1841.",
  "Abbedval i Vkl" => "Ordning vid abbedisseval i Vadstena kloster. I Småst på Fsv.",
  "AktKungsådr" => "Aktsamling till kungsådreinstitutets historia utg. av K. G. Westman. 1920.",
  "Annerstedt Ups Univ hist Bih 1" => "Upsala Universitets historia. Bihang I. Handlingar 1477--1654. Utg. af Claes Annerstedt. 1877.",
  "Arfstv" => "Arfstvisten emellan Erik Eriksson (Gyllenstjerna) och Ture Turesson (Bjelke) 1451--1480. Utg. af K. H. Karlsson. 1908. SFSS.",
  "Arkiv för Norrländsk hembygdsforskning." => "1918. s. 102 ff. En medeltida anteckningsbok från Stöde, utg. av Henrik Cornell.",
  "Arnell Brask" => "E. Arnell, Bidrag till biskop Hans Brasks lefnadsteckning. 1904. [Av de särskilt paginerade textbilagorna är I, s. 7--37, avtryckt ur hds. fr. 1513--27.]",
  "ATb" => "(el. ATb 1), ATb 2, ATb 3 Arboga stads tänkebok I--III. Utg. av Erik Noreen och Torsten Wennström. 1935--40. SFSS.",
  "Beckman Stud" => "N. Beckman, Studier i outgivna fornsvenska handskrifter. 1917. SFSS.",
  "Bergius Nytt förråd" => "Nytt förråd af äldre och nyare handlingar rörande nordiska historien. Utg. af B. Bergius. 1753.",
  "Bergqvara" => "B. Schlegel, Bergqvara gods och slott i Småland. 1897.",
  "Bergström, Arb Kr" => "Arboga Krönika. I. Medeltidsminnen. Efter samtida källor af G. Bergström. 1892.",
  "Bisk Nils" => "Biskop Nils visitationsstadga för Vadstena kloster af jan. 1448. I Småst på Fsv.",
  "Bjärka-Säby" => "H. Rosman, Bjärka-Säby och dess ägare. 1923.",
  "Brasks Cal" => "Calendarium oeconomicum avskrivet ur en gammal på Linköpings bibliotek förvarad hushållsbok (fr. 1513--27) efter biskop Brask. Efter avskrift av Per Kylander tryckt i Linköpings bibl:s handl. I s. 263--74 (1793).",
  "Brasks Kopieb" => "Biskop Hans Brasks kopiebok. Bilaga till Östergötlands Fornminnesförenings Tidskrift. I. 1875.",
  "Brasks Matordn" => "H. Hildebrand, Matordningen i biskop Hans Brasks hus (ur hds. fr. 1513--27). I Vitt. Hist. o. Ant. Ak:s Månadsblad 1885 s. 1--21.",
  "Delag Arch" => "De la Gardiska Archivet eller handlingar ur grefliga De la Gardiska biblioteket på Löberöd. Utg. av P. Wieselgren. Del. 1--20. 1831--44.",
  "Dipl Norcop" => "Norrköpings Medeltid. Ett Diplomatorium Norcopense samlat och bearbetat av A. Nordén. 1918.",
  "EK" => "Erikskrönikan enligt Cod. Holm. D 2 jämte avvikande läsarter ur andra handskrifter utg. av R. Pipping. 1921. SFSS.",
  "Fem Moseb ed Thorell" => "Fem Moseböcker på fornsvenska enligt Cod. Holm. A1 utg. av O. Thorell. SFSS.",
  "Flores ed. Olson" => "Flores och Blanzeflor. Kritisk upplaga utg. av Emil Olson. 1921. SFSS.",
  "FMU" => "Finlands medeltidsurkunder. Utg. af Finlands Statsarkiv genom R. Hausen. 1--5. 1910--28.",
  "FOVkl" => "Tvänne stadgar rörande förbättrad observans inom Vadstena kloster. I Småst på Fsv.",
  "FragmBir ed Gussgard" => "To fragmenter på svensk av den hellige Birgittas skrifter av Jostein Gussgard. 1961. SFSS.",
  "Fredrik ed Noreen" => "Hertig Fredrik av Normandie. Kritisk upplaga utg. av Erik Noreen. 1927. SFSS.",
  "Fyra handl rör Vkl" => "Fyra handlingar rörande Vadstena klosters privilegier (1440 och 1458), abbedissans årliga redovisning och klosterfolkets tarf af kost och föda. I Småst på Fsv.",
  "Gadolin Pants" => "A. W. Gadolin, Pantsättning af jord enligt medeltida svensk landsrätt. Med en bilaga af förut otryckta urkunder från perioden 1351--1400. 1909.",
  "GPM" => "Gamla papper angående Mora socken. II. Arvid Siggessons brevväxling. Utg. av L. Sjödin och A. Pers. 1932.",
  "GU C 20" => "Latinskt-Svenskt glossarium efter Cod. Ups. C 20. Utg. av E. Neuman. S 1--583. 1918--20, (hand 2) s. 1--169. 1938--42. SFSS.",
  "Gummerus Syn-stat" => "Synodalstatuter och andra kyrkorättsliga aktstycken från den svenska medeltidskyrkan. Utg. af J. Gummerus. 1902.",
  "Gyll år" => "Den fornsvenska dikten Om ett gyllene år. Utg. av G. Blomqvist. 1970.",
  "Hedqvist Kärleksv" => "Den kristna kärleksverksamheten i Sverige under medeltiden. Ak. afh. af Vilhelm Hedqvist. 1893.",
  "Hel män" => "Helige mäns lefverne jämte legender och järtecken. Utg. af R. Geete. 1902. SFSS.",
  "Hist Handl" => "Historiska Handlingar till trycket befordrade af K. Samf. f. utg. af hdskr. rör. Skandinaviens historia. 1861 ff.",
  "HLG 1" => "Handlingar rörande Helga Lekamens Gille i Stockholm. I. Gillesboken 1393--1487. Utg. av I. Collijn. 1921.",
  "HLG 2" => "Handlingar rörande Helga Lekamens Gille i Stockholm. II--IV. Erogata 1509--1528. Utg. av I. Collijn. 1923.",
  "HLG 3" => "Handlingar rörande Helga Lekamens Gille i Stockholm och Helgeandshuset i Uppsala. V--VIII. Gillets leuata 1515--1528. -- Räkenskaper för Helgeandshuset i Uppsala 1528--1529. -- Utg. av I. Collijn. 1930.",
  "Inv cur Tynnelsö" => "Inventarium curiæ Tynnelsö anno 1443. Ed. J. H. Schröder. 1839.",
  "HT" => "Historisk Tidskrift.",
  "Ivan ed. Noreen" => "Herr Ivan. Kritisk upplaga utg. av Erik Noreen. 1930 o. följ. SFSS.",
  "J Buddes b" => "Jöns Buddes bok. Utg. genom O. F. Hultman. 1895.",
  "JPMs" => "Jungfru Marie Psaltare. Utg. av R. Geete. 1923--25. SFSS.",
  "JMÖ" => "Jungfru Marie Örtagård. Vadstenanunnornas veckoritual i svensk öfversättning från år 1510. Utg. af R. Geete. 1895. SFSS.",
  "JTb" => "Jönköpings stads Tänkebok från 1456--1548. Utg. af C. M. Kjellberg. 1910--19. Bilaga till Meddelanden från Norra Smålands Fornminnesförening h. 2--5.",
  "JämtlDipl" => "Jämtlands och Härjedalens diplomatarium utg. under red. av K.-E. Löfqvist o. R. Swedlund. 1943 ff.",
  "KarlM (Ver)" => "och KarlM (El) Karl Magnus enligt Codex Verelianus och Fru Elins bok. Utg. av D. Kornhall. 1957. SFSS.",
  "KarlM (El)" => "/se KarlM (Ver)/",
  "KTb" => "Kalmar stads tänkebok. Utg. av I. Modéer och S. Engström. 1945. SFSS.",
  "Kumla kyrkas rb" => "Kumla kyrkas räkenskapsbok 1421--1590. Utg. av J. L:son Samzelius. 1946.",
  "Kyrkohist Årsskr" => "Kyrkohistorisk Årsskrift. 1900 ff.",
  "Lagerbring Saml" => "Lagerbring, Samling af åtskilliga handlingar och påminnelser, som förmodeligen kunna gifva ljus i Svenska Historien. Del. 1--3. 1754--58.",
  "Liber eccl Vallentunensis" => "Liber ecclesiæ Vallentunensis utg. av T. Schmid. 1945.",
  "Linkbiblh" => "Linköpings bibliotheks handlingar. 1--2. 1793--5, ny ser. 1 ff. 1920 ff.",
  "Mecht" => "Hel. Mechtils uppenbarelser (Liber spiritualis gratiæ) öfversatta från latinet år 1469 af Jöns Budde. Utg. af R. Geete. 1899. SFSS.",
  "Meddelanden från Östergötlands Fornminnes- och Museiförening 1916" => ", s. 91 ff. Norrköpings två äldsta privilegiebrev utg. av Carl M. Kjellberg.",
  "Mon Dipl Sv" => "Monumenta diplomatica Svecana. Ed. J. H. Schröder. 1820--22.",
  "MPFr" => "Fragmente einer mittelschwedischen Postille aus Barmen. Till trycket befordrad av R. Meissner. Arkiv för Nord. Fil. 39.",
  "MUKopp" => "Medeltida urkunder rörande Stora Kopparberget. Utg. av E. Wessén. 1947.",
  "Namnlös ed. Wolf" => "Namnlös och Valentin. Kritische Ausgabe. Herausgeg. von Werner Wolf. SFSS.",
  "Neuman Vokbal" => "E. Neuman, Utbredningen av vokalbalansen a:å i medelsvenskan. 1918.",
  "Nio handl rör Vkl" => "Nio handlingar rörande Vadstena klosters fridlysta område, dess inhägnad, tomter och byggnad samt frihet från intrång. I Småst på Fsv.",
  "Nio kap ur Bir" => "Nio kapitel ur H. Birgittas uppenbarelser nedskrifna omkr. år 1385. I Småst på Fsv.",
  "NMU" => "Närkes medeltida urkunder I. Riseberga kloster. Utg. av K. G. Grandinsson. 1935.",
  "Nunneinv" => "Ceremonielet vid nunneinvigning i Vadstena kloster. I Småst på Fsv.",
  "Ol. Magnus Hist. de gent. septenir." => "Historia de gentibvs septentrionalibvs ... Avtore Olao Magno Gotho ... 1555.",
  "PMBref" => "Vadstenabrodern Peder Månssons bref på svenska från Rom till Vadstena kloster 1508--1519. I Småst på Fsv.",
  "PMSkr" => "Peder Månssons Skrifter på svenska. Utg. af R. Geete. 1913--15. SFSS.",
  "Priv f Sv st" => "Privilegier, resolutioner och förordningar för Sveriges städer. Första delen (1251--1523) utg. av N. Herlitz. 1927.",
  "Priv o skyddsbr" => "Privilegie- och skyddsbref för Vadstena kloster utfärdade af K. Kristofer åren 1442 och 1444. I Småst på Fsv.",
  "Prosadikter" => "Prosadikter från Sveriges medeltid. Utg. af G. E. Klemming. 1887--89. SFSS.",
  "Reg. Eccl. Ab." => ", se Svartb.",
  "Reuterdahl Kyrkohist" => "H. Reuterdahl, Svenska kyrkans historia. III, 2:528--59.",
  "Reuterdahl Stat syn" => "Statuta synodalia veteris ecclesiæ Sveogoticæ Ed. H. Reuterdahl. 1841.",
  "RP" => "Svenska Riks-archivets pergamentsbref från och med år 1351. Förtecknade med angifvande af innehållet. 1, 2. 1866--68.",
  "Rydberg Tr" => "Sveriges traktater med främmande magter jemte andra dit hörande handlingar. Utgifna af O.S. Rydberg. Del. 2, 3. 1880--95.",
  "Saml" => "Samlaren. Tidskrift utgifven af Svenska Literatursällskapets arbetsutskott. 1880 o. följ.",
  "Schlegel o. Klingspor" => "B. Schlegel och C. A. Klingspor, Den med sköldebref förlänade men ej på riddarhuset introducerade svenska adelns ättartaflor. 1875.",
  "Scr. rer. svec." => "Scriptores rerum svecicarum medii aevi. 1818 o. följ.",
  "Sex ekon tr" => "Om trädplantering, beredning af mjöd, claret och salvieöl, matsedlar från olika land samt om brännvins nytta. Sex ekonomiska traktater från medeltiden. I Småst på Fsv.",
  "SFSS" => "Samlingar utgifna af Svenska Fornskriftssällskapet.",
  "Skotteb" => "Stockholms stads skottebok 1460--1468 samt strödda räkenskaper från 1430-talet och från åren 1460--1473. Utg. genom J. A. Almquist. 1926. [Beteckningen Borgm avser de borgmästareräkenskaper som äro tryckta s. 337--356 och Kämn kämnärsräkenskaperna s. 359--480].",
  "SkrtUppb" => "Skrifter till uppbyggelse från medeltiden. Utg. af R. Geete. 1904--05. SFSS.",
  "Småst på Fsv" => "Småstycken på Forn Svenska. Andra serien. Samlad af R. Geete. 1900-1916. SFSS.",
  "Spegel Skrbev" => "Skriftelige bevis hörande till svenska kyrckiohistorien eller biskops chrönikan [af Haqvin Spegel]. Ups. 1716.",
  "SpV" => "Speculum Virginium -- Jungfruspegel -- öfvers. från latinet af Mathias Laurentii. Utg. af R. Geete. 1897--98. SFSS.",
  "SSkb" => "Stockholms Stads Skottebok 1501--1510. Utg. genom H. Hildebrand. 1889, 1915. SSKb 2:= Stock Skb",
  "Stb 1" => "Stockholms Stads Tänkeböcker 1474--1483 samt Burspråk. Utg. genom E. Hildebrand. 1917.",
  "Stb 2" => "Stockholms Stads Tänkeböcker 1483--1492. Utg. genom G. Carlsson. 1921--44.",
  "Stb 3" => "Stockholms Stads Tänkeböcker 1492--1500. Utg. genom J. A. Almquist. 1930.",
  "Stb 4" => "Stockholms Stads Tänkeböcker 1504--1514. Utg. genom J. A. Almquist. 1931.",
  "Stb 5" => "Stockholms Stads Tänkeböcker 1514--1520 jämte utdrag ur de förlorade årgångarne 1520--1524 samt stadens kopiebok 1520--1522. Utg. genom J. A. Almquist. 1933.",
  "Stock Skb" => "Stockholms stads skottebok 1516--1525 samt strödda räkenskaper. Utg. genom J. A. Almquist. 1935.",
  "StÄmb" => "Stockholms Stads Ämbetsbok 1419--1544. Utg. genom J. A. Almquist. 1927.",
  "Svartb" => "och Svartb (Skokl) Registrum Ecclesiæ Aboensis eller Åbo domkyrkas Svartbok med tillägg ur Skoklosters Codex Aboensis. Utg. genom R. Hausen 1890.",
  "Svartb (Skokl)" => "/se Svartb/",
  "SvB" => "Svenska Böner från medeltiden. Utg. af R. Geete. 1907--09. SFSS.",
  "SvGst" => "Fragment af en svensk gillesstadga. I Småst på Fsv.",
  "SvKyrkobr" => "Svenska Kyrkobruk under medeltiden. Utg. af R. Geete. 1900. SFSS.",
  "SvRimd" => "Två Svenska Rimdikter från medeltiden. I Småst på Fsv.",
  "SvSkr" => "Svenska Skriftprof ... i boktryck återgifna af E. Hildebrand, A. Börtzell och H. Wieselgren. H. 1. 1894.",
  "SVklE" => "Stadga af år 1443 för Vadstena klosters ekonomi. I Småst på Fsv.",
  "SvT" => "Den Svenska Tideboken. [Utg. av G. E. Klemming] 1854. [Efter gammalt tryck fr. c. 1525].",
  "Thomas Varningsbref" => "Varnings-bref från Biskop Thomas i Strengnäs till Konung Carl Knutsson. 1837.",
  "Troil" => "Skrifter och handlingar til uplysning i svenska kyrko och reformationshistorien. [Utg. av U. von Troil.] 1--5. 1790--91.",
  "Troj" => "Historia Trojana. Från latinet öfversatt till svenska år 1529. Utg. af R. Geete. 1892. SFSS.",
  "Trolles Jb" => "Arvid Trolles jordebok 1498 jämte åtkomsthandlingar och andra därmed samhöriga aktstycken. Utg. av J. A. Almquist. 1938.",
  "Tunberg Sv medeltidsreg." => "Svenska medeltidsregester. Förteckning över urkunder till Sveriges historia 1434--1441. Utg. av Sven Tunberg ... 1937.",
  "Uppl Fornmt" => "Upplands Fornminnesförenings tidskrift.",
  "Uppl Lagmansdomb" => "Upplands Lagmansdombok 1490--1494. Utg. af K. H. Karlsson. 1907. SFSS.",
  "Uppspriv" => "Uppsala stads stadsprivilegier jämte dit hörande handlingar. Utg. af C. M. Kjellberg. 1907.",
  "UrkShist" => "Urkunder rörande Stockholms historia. Utg. genom K. Hildebrand (och A. Bratt). 1900--1913. (s. 1--36).",
  "Val af Conf gen" => "Ordning vid val af confessor generalis i Vadstena kloster. I Småst på Fsv.",
  "Vg Fornmt" => "Vestergötlands Fornminnesförenings tidskrift. 1869 o. följ.",
  "Vis sten" => "Den vises sten. I Småst på Fsv.",
  "VKJ" => "Vadstena klosters Jordebok 1500 jemte tillägg ur klostrets äldre jordeböcker. Utg. genom C. Silfverstolpe. 1898. [Beträffande dateringen jfr E. Noreen, Historisk Tidskrift 39, s. 187 f.]",
  "VKU" => "Vadstena klosters Uppbörds- och Utgiftsbok 1539--1570. Utg. af C. Silfverstolpe. 1895.",
  "Vm Fornm Årsskr" => "Vestmanlands Fornminnesförenings årsskrift. 1874 o. följ.",
  "Vrml skatteb" => "En värmländsk skattebok från 1503. Utg. av S. Samuelsson och G. Kallstenius. Nationen och hembygden 2.",
  "Vårfrup" => "Handlingar på svenska rörande 'Vårfrupänningen' till Vadstena klosters byggnad och underhåll. I Småst på Fsv. Växtförteckn. fr. slutet af 1400-talet hos Th. M. Fries, Ark. f. Bot. Bd 3 nr 14. Växtförteckn. fr. medlet af 1400-talet hos Th. M. Fries, Ark. f. Bot. Bd 3 nr 14.",
  "ÅK" => "Åbo Klockarelag. I Småst på Fsv.",
}
`;
